lib/modules/apostrophe-events-calendar-widgets/public/js/always.js

```diff
@@ -0,0 +1,38 @@
+apos.define('apostrophe-events-calendar-widgets', {
+  extend: 'apostrophe-pieces-widgets',
+  construct: function(self, options) {
+    self.play = function($widget, data, options) {
+
+    	var cal;
+
+			function getCalendar(data, callback) {
```

Comment:
Copy in .eslintrc and the usual eslint suspects in package.json, and include `eslint` in the npm test config, so whitespace stuff like this is easy to clean up.

Reply:
(You can borrow the setup that's in the apostrophe module, it should work well here.)